dvc/data/reference.py

```diff
@@ -57,11 +59,16 @@ def to_bytes(self):
         # ReferenceHashFiles should currently only be serialized in
         # memory and not to disk
         fs_path = self.fs_path
+        fs_cls = type(self.fs)
+        mod = None
+        if fs_cls not in FS_MAP:
+            mod = ".".join((fs_cls.__module__, fs_cls.__name__))
```

Comment:
Will only use this if it is not in FS_MAP, i.e. not implemented in dvc.objects.fs.